### PR TITLE
Nuke Ops weapon tweaks

### PR DESCRIPTION
**Pull request by Kierany9**
Minor nuke ops weapon changes that remove bloat and hopefully make these weapons more useful.

### Sniper Rifle (14 TC)
- **Sniper Rifles now dismember targeted limbs when using standard or high-explosive ammo.** Can't remove the head or chest for obvious reasons.
- **Replaced haemorrhage rounds with high-explosive rounds.** These deal 40 damage with no armour penetration(as opposed to 70 damage and 50 AP) but explode in a 1-tile radius(3x3 area) similar to the gyrojet pisol. Like regular rounds, these can delimb and break through walls.
- All sniper magazines now hold five shots regardless of type (Standard held 6, Sleepy held 3, Penetrator held 5).
- Sniper bundle now contains a HE magazine and costs 2 TC more (20 -> 22)
- Decreased the price from 16 TC to 14 TC. However, the price of ammo has gone up (Standard: 4 -> 5, Sleepy: 5 -> 3, High Explosive: 6, Penetrator: 5 -> 6)

### L6 SAW (23 TC)
- **Reverts the nerf to its firepower, bullets now deal 60 damage instead of 35.** Now also has 10 armour penetration.
- **Removed all the special ammo types.** There was no practical use for any of these. Haemorrage and Incendiary had laughable damage outputs(5 and 15 per shot respectively) whereas AP and hollow were simpe stat variations that changed very little when compared to standard rounds.
- Ammo decreased from 6 TC to 5 TC

### Abzats (27 TC)
- Decreased price from 30 TC to 27 TC to compensate for the SAW buff.
- Decreased the price of its spare ammo. (40-round buckshot box: 8 -> 7, 40 round dragonsbreath box: 12 -> 9, 40-round buckshot magazine: 10 -> 8)
- **Fixed a bug where 40-round buckshot boxes could not be purchased from the uplink.**

### CODE REVIEW

**Comment by calibraptor** [comment]
You should be wrecking face if you pool enough TC to purchase these anyways so, why the hell not?

+1/3rd

**Comment by trifornt-tinfornt** [comment]
Did the L6 Saw's crappy "kicks out of your hand!" mechanic get removed? The Abzats is even MORE powerful and it doesn't have that.

**Comment by McBawbaggings** [comment]
I'm scared about adding a dismembering weapon but if you're getting shot by syndie snooper rifle then you're dead anyway. So okay, since it doesn't INSTA DECAP it's fine.
I've never seen ammotypes used so sure why not.
#MakeSawGreatAgain
So the point of sleeping bullets is they're effectively a hit kill? Wouldn't that make sense they're more expensive and you get less? (Although the whole 2 hit kill from snipers is probably strong enough).
TC costs change seem alright I guess?

I don't outright object to this PR but I'd like a bit more reasoning behind the cost changes and the sniper rifle knockout changes before I give it the sweet +1.

**Comment by Carbonhell** [comment]
wait,if i shoot with the sniper rifle loaded with sleepy bullets..will i still delimb the guy
it makes no sense if the reagents go in the arm but the arm gets dismembermembermbemhow the fuck do i fucking spell this word god fucking damnit
anyway
nice pr

**Comment by Kierany9** [comment]
>Sniper Rifles now dismember targeted limbs when using standard or high-explosive ammo. Can't remove the head or chest for obvious reasons.
>Sniper Rifles now dismember targeted limbs when using standard or high-explosive ammo.
>when using standard or high-explosive ammo

reading is fun

**Comment by Carbonhell** [comment]
reading at 2:30 am is not fun nigger

**Comment by McBawbaggings** [comment]
After some discussion, I see the reason in these changes.
Hopefully this will change the good auld strat of buying that stunny SMG thingy.

**Comment by Badmex** [comment]
Are people really buying the other ammo for the sniper rifle?
Put yeah overall it does sound good +1/3

**Comment by dreamcrab** [comment]
i thought i posted on this +2515

**Comment by Kierany9** [comment]
Still needs a codecheck.

...

...

@carbonhell

File: html/changelogs/Kierany9 - Newcops.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+author: Kierany9
+
+delete-after: True
+
+changes: 
+- rscadd: "Nuke Ops Sniper Rifles can now remove targeted limbs if they're using standard or HE ammunition!"
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - Newcops.yml]: wrong indentation fam

**Comment by Carbonhell** [comment]
Code's ok,merging

**Pull request merged**